sites/game-client/ui/overlay.mjs

```diff
@@ -39,6 +43,20 @@ class Overlay {
     return true;
   }
 
+  /**
+   * If true, it should set the basic property for toolbarButton,
+   * and mainUI will add a button in toolbar for the overlay.
+   *
+   * *This method can be implemented in the derived class if the derived class
+   * wishes to adjust this default behaviour.*
+   *
+   * @return {Boolean}
+   */
+  hasToolbarbutton() {
```

Comment:
Should be `hasToolbarButton`. (**B**utton, capital B)

Reply:
resolved in c0e3bc3